sandy/sections/mf10.py

```diff
@@ -0,0 +1,199 @@
+    >>> sec = read_mf10(tape, 4125, 16)
+    >>> text = write_mf10(sec)
+    >>> print(text)
+     41093.0000 92.1082700          0          0          2          0412510 16    1
+    -8830560.00-8830560.00      41092          0          1         31412510 16    2
+             31          2                                            412510 16    3
+     8926430.00 0.00000000 9000000.00 1.941860-2 9500000.00 1.621040-1412510 16    4
+     10000000.0 3.089020-1 10500000.0 4.351160-1 11000000.0 5.174740-1412510 16    5
+     11500000.0 5.739980-1 12000000.0 6.246930-1 12500000.0 6.593280-1412510 16    6
+     13000000.0 6.818000-1 13500000.0 6.988180-1 14000000.0 7.179610-1412510 16    7
+     14500000.0 7.367761-1 15000000.0 7.507430-1 16000000.0 7.747931-1412510 16    8
+     17000000.0 7.894241-1 18000000.0 7.807111-1 19000000.0 7.326431-1412510 16    9
+     20000000.0 6.574070-1 21000000.0 5.754551-1 22000000.0 4.940910-1412510 16   10
+     23000000.0 4.301710-1 24000000.0 3.861880-1 25000000.0 3.484020-1412510 16   11
+     26000000.0 3.224040-1 27000000.0 2.991570-1 28000000.0 2.820340-1412510 16   12
+     29000000.0 2.640060-1 30000000.0 2.530630-1 30000000.0 0.00000000412510 16   13
+      200000000 0.00000000                                            412510 16   14
+    -8830560.00-8966060.00      41092          1          1         30412510 16   15
+             30          2                                            412510 16   16
+     9063400.00 0.00000000 9500000.00 3.129770-2 10000000.0 1.272110-1412510 16   17
+     10500000.0 2.131390-1 11000000.0 2.852680-1 11500000.0 3.443660-1412510 16   18
+     12000000.0 3.896360-1 12500000.0 4.214350-1 13000000.0 4.415390-1412510 16   19
+     13500000.0 4.529980-1 14000000.0 4.591280-1 14500000.0 4.601300-1412510 16   20
+     15000000.0 4.603000-1 16000000.0 4.520400-1 17000000.0 4.450740-1412510 16   21
+     18000000.0 4.207860-1 19000000.0 3.720790-1 20000000.0 3.237210-1412510 16   22
+     21000000.0 2.723630-1 22000000.0 2.328870-1 23000000.0 2.067260-1412510 16   23
+     24000000.0 1.869740-1 25000000.0 1.722300-1 26000000.0 1.604680-1412510 16   24
+     27000000.0 1.507150-1 28000000.0 1.421910-1 29000000.0 1.353680-1412510 16   25
+     30000000.0 1.291160-1 30000000.0 0.00000000  200000000 0.00000000412510 16   26
+    """
+    lines = sandy.write_cont(
+            sec["ZA"],
+            sec["AWR"],
+            sec["LIS"],
+            0,
+            len(sec["LFS"]),
+            0,
+            )
+
+    for hz in range(len(sec["LFS"])):
+        LFS = sec["LFS"]
+        key = list(LFS.keys())
+        lines += sandy.write_tab1(
+                LFS[key[hz]]["QM"],
+                LFS[key[hz]]["QI"],
+                LFS[key[hz]]["IZAP"],
+                key[hz],
+                LFS[key[hz]]["NBT"],
+                LFS[key[hz]]["INT"],
+                LFS[key[hz]]["E"],
+                LFS[key[hz]]["XS"],
+                )
```

Comment:
this for loop should be written using the `.items()` option of dictionaries.
```
    for LFS, subsection in sec["LFS"].items():
        lines += sandy.write_tab1(
                subsection["QM"],
                subsection["QI"],
                subsection["IZAP"],
                LFS,
                subsection["NBT"],
                subsection["INT"],
                subsection["E"],
                subsection["XS"],
                )
```
Can you check that MF9 is also written this way? Thanks!